src/test/java/tests/TestRegister.java

```diff
@@ -21,12 +22,24 @@ public void TC07() {
         Log.info("> Create fake credentials");
         String email = DataHelper.getRandomEmail();
         String password = DataHelper.getRandomPassword();
-        String pid = DataHelper.getRandomPIDNumber();
 
         Log.info("> Register a new account with valid credentials");
-        registerPage.register(email, password, password, pid);
+        registerPage.register(email, password, password, Constant.DEFAULT_PID);
 
         Log.info("> Verify the successful message.");
         Assert.assertEquals(registerPage.getSuccessMsg(), "You're here", "Success message does not match!");
     }
+
+    @Test(description = "User can't create account with 'Confirm password' is not the same with 'Password'")
+    public void TC10() {
+        Log.startTestCase("TC10");
+        Log.info("> Go to Register page");
+        homePage.goToRegister();
+
+        Log.info("> Register with 'Confirm password' is not the same with 'Password'");
+        registerPage.register(Constant.NOT_REGISTERED_EMAIL, "12345678", "23456789", Constant.DEFAULT_PID);
```

Comment:
@InternDN21-DoHoaiLinh same above for variable declaration. Don't put hard code like this

Reply:
I fixed it. Thank you.